add custom consul config params

123BLiN opened the pull request:
add custom config params with choco -params directive

123BLiN committed:
add custom consul config params

add custom config params with choco -params directive

calvn commented on consul/consul.nuspec:
```diff
@@ -12,7 +12,12 @@
-Uses NSSM as the service wrapper. More info at: https://nssm.cc/</description>
+Uses NSSM as the service wrapper. More info at: https://nssm.cc/
+
+#### Package Parameters
```
I think it would be best if this goes to the Readme instead.

123BLiN replied:
I've added parameters usage info in description because of official recomendation to add it there :) But please feel free to remove. Also added to README.MD

calvn replied:
Can you link me the docs to this? I'd like to read about it :)

123BLiN replied:
https://github.com/chocolatey/choco/wiki/How-To-Parse-PackageParameters-Argument#add-package-parameter-information-to-the-description

123BLiN committed:
fix parameters usage info

update README.MD
fix info in description (will be shown at choco consul web page)

calvn commented:
@123BLiN sort of a **_very_** late response, but can you pull this PR into https://github.com/calvn/chocolatey-consul? I'd be glad to merge it there since we are looking into this feature at the moment.

123BLiN commented:
Hello @calvn !
I will try to make a PR in a couple of days, just need to find some time.

calvn commented:
@123BLiN Hey, no problem! I ported over your changes in a recent PR since there was a breaking bug for 0.7.4 package release.

123BLiN commented:
So we are good now?

calvn commented:
Yep, we should be good. Thanks for this PR btw!
